src/test/java/core/basesyntax/MainAppTest.java

```diff
@@ -1,5 +1,22 @@
 package core.basesyntax;
 
 public class MainAppTest {
+    public static void main(String[] args) {
 
+        Machine excavator = new Excavator();
```

Comment:
Use abstract references instead of specific ones where possible to take advantage of polymorphism. For example, `Machine excavator = new Excavator();` is correct, but ensure that you're using the `Machine` type for a reason and that polymorphic behavior is expected.